plugins/rapid7_intsights/plugin.spec.yaml

```diff
@@ -615,3 +615,114 @@ actions:
         description: New created alert ID
         type: string
         required: true
+triggers:
+  new_alert:
+    title: New Alert
+    description: Run when a new alert that matches the given criteria is created in Intsights
+    input:
+      alert_type:
+        title: Alert Type
+        description: 'List of alert types. Allowed values: AttackIndication, DataLeakage, Phishing, BrandSecurity, ExploitableData, vip'
+        type: '[]string'
+        required: false
+        example: ["Phishing"]
+      severity:
+        title: Severity
+        description: 'List of alerts severity. Allowed values: High, Medium, Low'
+        type: '[]string'
+        required: false
+        example: ["Low"]
+      source_type:
+        title: Source Type
+        description: 'List of alerts source type. Allowed values: Application Store, Cyber Security Blog, Hacking News, Cyber Crime Forum, Hacktivism Forum, Social Media, Facebook, Twitter, LinkedIn, Google Plus, VK, Vimeo, YouTube, IRC Channel, IOC Block List, Credit Card Black Market, Paste Site, Data Leakage Website, Leaked Database, File Sharing Website, Gray Hat Website, Black Market, WHOIS servers, Company Website, Wikileaks, Pinterest, Tumblr, Instagram, Telegram, Webmail, Malware Analysis, Firehol, VRA, Other'
+        type: '[]string'
+        required: false
+        example: ["Application Store"]
+      network_type:
+        title: Network Type
+        description: 'List of network type. Allowed values: ClearWeb, DarkWeb'
+        type: '[]string'
+        required: false
+        example: ["DarkWeb"]
+      matched_asset_value:
+        title: Matched Asset Value
+        description: List of matched asset values
```

Comment:
Can we give any more info to the user here? A new user might not know what to input here

Reply:
I changed here description, but in documentation there is no more information about this field. Could we contact with intsight team and ask which value can be here?